figo/figo.py

```diff
@@ -577,6 +575,29 @@ def remove_account(self, account_or_account_id):
 
         return None
 
+    def sync_account(self, state, redirect_uri, account_ids, if_not_synced_since, sync_tasks=['transactions'],
+                     disable_notifications=False, auto_continue=False):
+        """
+        Trigger bank account sync
+        
+        :Parameters:
+         - `account` - account ID to be synced
+          
+        :Returns:
+         - `task_token` - task token
+        """
+        data = {'state': state,
+                'redirect_uri': redirect_uri,
+                'disable_notifications': disable_notifications,
+                'if_not_synced_since': if_not_synced_since,
+                'auto_continue': auto_continue,
+                'account_ids': account_ids,
+                'sync_tasks': sync_tasks}
+
+        task_token = self._query_api_object(model=TaskToken, path='/rest/sync', data=data, method='POST')
```

Comment:
According to the above, add

```python
data = dict((k, v) for k, v in data.tems() if v is not None)
```

to remove unset arguments. This is obviously a pattern in the existing code and should be refactored at some point...